examples/AspNet/Controllers/WeatherForecastController.cs

```diff
@@ -181,7 +181,8 @@ private async Task RequestInvalidViaHttpClient()
 
             // This request is not available over SSL and will throw a handshake exception.
 
-            using var response = await request.GetAsync(this.Url.Content("~/subroute/10").Replace("http", "https")).ConfigureAwait(false);
+            using var response = await request.GetAsync(
+                new Uri(this.Url.Content("~/subroute/10").Replace("http", "https"))).ConfigureAwait(false);
```

Comment:
nit: Probably good to extract some of these statements into their own variables.

Reply:
Updated